examples/systemcalls/systemcalls.c

```diff
@@ -17,7 +23,15 @@ bool do_system(const char *cmd)
  *   or false() if it returned a failure
 */
 
+    if (cmd == NULL)
+        return false;
+
+    int status = system(cmd);
+
+    if (status == -1)
```

Comment:
@bjornhnelson think about what could happen here if the command returned an error value other than -1.